### ISSUE TITLE
Error when running Hardware_Acceleration/Introduction/guided_sw_examples tutorial

### ISSUE THREAD

**Issue by Casper-Min** [Xilinx/Vitis-Tutorials#152]
Hi, I got error when running guided_sw_examples tutorial using U250, vitis 2021.1 on CentOS7 server. 

![image (2)](https://user-images.githubusercontent.com/83194777/148931174-9ba0efa2-57c0-46e6-9b9a-0028d057bf61.png)

From the log, I found that m_axi port bundle name doesn't match between connectivity_u250.ini and wide_vadd.cpp

![image](https://user-images.githubusercontent.com/83194777/148931683-4e76610d-88b6-4029-bb95-68b61a87062a.png)

![image](https://user-images.githubusercontent.com/83194777/148931276-c4e3be47-923f-428c-a70b-9f5fe4681b12.png)

(Also only 1 sp for vadd.cpp is describe on connectivity_u250.ini. shoud I insert additional two sp for vadd.cpp ? )

![image](https://user-images.githubusercontent.com/83194777/148931814-a1afd157-1842-4485-a82a-c3f909ff4d8b.png)

After I modified connectivity_u250.ini, I passed error above but still got error for vpl. 

![image](https://user-images.githubusercontent.com/83194777/148932202-381e0905-4873-4e55-8211-8198547a1da8.png)

Do you guys have any idea what is wrong with my setting or tutorial?

Thanks,
Casper

**Comment by Casper-Min** [comment]
And I also have question for using vitis vision library.

![image](https://user-images.githubusercontent.com/83194777/148932818-85372716-3766-465b-abac-f2cd1bd8c92e.png)

 First I got errors for missing header (common/xf_common.hpp), I couldn't find way to import this header, I git cloned Vitis_Libraries(https://github.com/Xilinx/Vitis_Libraries) on hw_src folder.
After that, I passed error above but I am not sure it is right way to solve error.

As many examples on Hardware_Acceleration/Design_Tutorials need Vitis_Libaray, locating Vitis_Libararies on all tutorial folder is impossible. Is there any proper ways to use Vitis_Libararies among tutorials?

Thanks,
Casper

**Comment by allyzhou** [comment]
@randyh62 can you take a look at this issue, Thanks.

**Comment by randyh62** [comment]
Hi Casper, The tutorials uses the Vitis_Libraries/vision/L1/include as indicated in the design_sources/hw_src/Makefile:
`VISION_LIB_FLAGS := -I. -I./Vitis_Libraries/vision/L1/include --config vision_config.ini`

The installation and setup instructions are kind of vague (missing) on this point, but you can find the required file here:
[https://github.com/Xilinx/Vitis_Libraries/tree/master/vision/L1/include/common](https://github.com/Xilinx/Vitis_Libraries/tree/master/vision/L1/include/common)

You can modify the Makefile in the hw_src folder to point to your Vitis_Libraries installation or the path to it. 

As to your VPL error, it could be a memory issue? Not sure what your system configuration looks like, but I don't see any other indication. I would suggest run it again and if you still have a problem, look for the Vivado log file which you should be able to find at: `hw_src/_x/link/vivado/vpl`

Also, can you post your modified `connectivity_u250.ini` so I can see what changes were needed to run the tutorial? 

Best regards,
Randy

**Comment by Casper-Min** [comment]
Hi, randyh62. Thank you for fast reply.

> You can modify the Makefile in the hw_src folder to point to your Vitis_Libraries installation or the path to it.

Modify Vitis_Libraries path is a good solution for further tutorial examples.

> I would suggest run it again and if you still have a problem

I did make clean and ran it again. I finished it successfully without any error after that.

> Also, can you post your modified connectivity_u250.ini so I can see what changes were needed to run the tutorial?

Below is my connectivity_u250.ini (which I finished build sucessfully)

![image](https://user-images.githubusercontent.com/83194777/150470406-15e338e7-d75a-4781-b6ad-aab2fdb71b4b.png)

I modified 2 parts.

1. I modified sp of wide_vadd bundle name according to wide_vadd.cpp

wide_vadd.cpp
![image](https://user-images.githubusercontent.com/83194777/150470509-d201d6b8-26ff-4654-ae7a-c71c6b88ab00.png)


2. I modified vadd.cpp and sp of vadd bundle name according to vadd.cpp

vadd.cpp baseline
![image](https://user-images.githubusercontent.com/83194777/150470827-057cafcc-786d-41a7-9102-9cf8eadca7ce.png)

I modified vadd.cpp Interface declear similar to wide_vadd.cpp
![image](https://user-images.githubusercontent.com/83194777/150470865-14e244c5-ba50-4254-b771-b0b01bf486ca.png)

Thanks,
Casper
